manifests/d/DBeaverCorp/DBeaverEnterpriseEdition/24.3.0/DBeaverCorp.DBeaverEnterpriseEdition.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: DBeaverCorp.DBeaverEnterpriseEdition
+PackageVersion: 24.3.0
+InstallerType: nullsoft
+InstallerSwitches:
+  Silent: /S /allusers
+  SilentWithProgress: /S /allusers

```

Comment:
Since `/allusers` is specified, the `scope` should also be specified. However, it would be preferred to have separate installer nodes with one specifying machine scope and `/allusers` and one specifying user scope and `/currentuser`